problems/objects.js

```diff
@@ -61,7 +71,10 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  sumOfAdamsAndEvesApples = appleCountByName.Adam + appleCountByName.Eve;
```

Comment:
_sumOfAdamsAndEvesApples_ was not defined as a local variable to this function, so it becomes global